### PR TITLE
CARRY: Update OWNERS file for openshift CI Job setup

### PR DESCRIPTION
**Pull request by ChughShilpa**

### CODE REVIEW

**Review by sutaakar — COMMENTED** [review]
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 
 reviewers:
-- alculquicondor
-- denkensk
-- kerthcet
-- mimowo
-- tenzen-y
-- trasc
-
-emeritus_approvers:
```
Comment (astefanutti) [inline OWNERS:15]: I think we can remove that section.

Comment (ChughShilpa) [reply OWNERS:15]: These are in remove part only

Comment (astefanutti) [reply OWNERS:15]: ah right, my mistake.

**Pull request merged**